actions/sequester/Quest2GitHub/AzDoClientServices/QuestClient.cs

```diff
@@ -42,34 +50,19 @@ public QuestClient(string token, string org, string project)
     /// <returns>The JSON packet representing the new item.</returns>
     public async Task<JsonElement> CreateWorkItem(List<JsonPatchDocument> document)
     {
-        var options = new JsonSerializerOptions
-        {
-            WriteIndented = true,
-            Converters =
-            {
-                new JsonStringEnumConverter(JsonNamingPolicy.CamelCase)
-            }
-        };
-        var jsonString = JsonSerializer.Serialize(document, options);
-        using var request = new StringContent(jsonString);
+        var json = JsonSerializer.Serialize(document, s_options);
+        Console.WriteLine($"Creating work items with:\n{json}");
```

Comment:
It's always singular:

```suggestion
        Console.WriteLine($"Creating work item with:\n{json}");
```